packages/pcd/eddsa-pcd/test/EDDSAPCD.spec.ts

```diff
@@ -40,4 +40,16 @@ describe("eddsa-pcd should work", function () {
     expect(deserializedValid).to.eq(true);
     expect(pcd).to.deep.eq(deserialized);
   });
+
+  it("should be able to compatibly deserialize a saved PCD", async function () {
+    const savedPCD =
```

Comment:
same suggestion for comment explaining how this was generated as I let in `EdDSAPCD.ts`